### DISCUSSION TITLE
Cannot access part of Wiki

### DISCUSSION THREAD

**Discussion by david-unwin** [akeeba/panopticon#316] [Q&A (Community Support)] [answered]
When I try and access the Backup Management or Security Wiki pages. I get the message. "_You do not have permission to update this wiki."_ All the other pages in the Wiki I can access....Any ideas what I am doing wrong???
![Home-·-akeeba-panopticon-Wiki](https://github.com/akeeba/panopticon/assets/95095238/061b16da-3391-4cab-a57f-544ad594436b)

**Comment by brianteeman** [comment, marked as answer]
That is because there is no content in those two pages yet
